website/docs/getting_started/conceptual_overview.md

```diff
@@ -0,0 +1,91 @@
+# Pact Conceptual Overview 
+Using Pact successfully requires you to understand how Pact works and the concepts and terminology behind Pact. 
+If you try to build your CI/CD without this foundational understanding, you may find yourself getting confused or getting unexpected results.
+
+## Consumers and publishing pacts
+Whenever a consumer runs a contract test for a particular provider and it succeeds, a file is generated that records 
+all of the interactions between the consumer and the provider. This record is called a pact.
+
+In your CI/CD environment, every time you run the contract tests, the resulting pact will be published to the Pact Broker. 
+This is called a pact publication.
+
+Whenever the consumer changes, this is called a __consumer version__. The version is often identified by its git hash. 
+You may also want to add other distinguishing information, such as the feature toggle name if you are using feature 
+toggles.  In general, you want to uniquely identify each version of the consumer which may require a different contract.
+
+A given pact published by the consumer may or may not change actual contents of the pact (e.g. by adding a request 
+parameter, or changing the expected results).  If it does change, then the pact gets a new __pact version__.
+
+![Consumer domain model](conceptual_overview_images/consumer-domain-model.png)
+
+It's important to understand a few rules about how consumers relate to pacts:
+
+- Multiple versions of the consumer can have the same pact version (if the changes to the consumer don't change the pact)
+- A specific version of the consumer will always have exactly one pact version
+
+## Tagging
```

Comment:
Do we want to go into tagging first, given it's now "deprecated" (or at least not best practice)? Or do we talk about branches and envs first and why we need them, then in a later section say "in old versions of the broker we didn't have branches and envs, so this is how we represented them"?

Reply:
Sure that's fine

Reply:
Have changed the term to be labelling with a branch, recording an environment, rather than mentioning tagging at all, bar a caveat with a section to how they can migrate to branches/environments. It is a bit of a rehash rather than a full rewrite atm.

WDYT